### DISCUSSION TITLE
The best alternative to LSPosed

### DISCUSSION THREAD

**Discussion by juvannx** [mywalkb/LSPosed_mod#46] [General]
Thank you for your work.
I tried other forks but this is the best.

The others make fake updates to make people believe that the project is maintained, but they are useless and it is not clear the reason for the update.

The others do not solve issues but you instead support android 15 while the others are still waiting for the original developers to make updates.

I tried your API to be able to modify the scope of a module, your way was simple and intuitive, the original developers rewrote the API and then abandoned the project, they advertised on xda making people believe who knows and it ended badly !

I hope you will continue your work.
CLI is good but you should improve it, for example I can't set "Automatic add" with cli, I hope to see updates.

**Comment by mywalkb** [comment]
> The others make fake updates to make people believe that the project is maintained, but they are useless and it is not clear the reason for the update.

I noticed it too

> I hope you will continue your work. CLI is good but you should improve it, for example I can't set "Automatic add" with cli, I hope to see updates.

I maintain it in my spare time, so I'm not quick to fix issues and implement new features. As for the CLI, if you have suggestions, open an FR.

**Reply by TGSAN** [reply]
I love CLI feature, thank you!

**Reply by mywalkb** [reply]
I'm glad you like it, I'm not the author of the CLI because I took it from a PR not accepted in the original repository and I improved it, the important thing is that it is useful to someone, I like to give the possibility to choose . For example, I don't use the CLI much except for testing and corrections, but I think it's right that it's there. If you have suggestions for improving it, they are welcome.

**Comment by BlueCat300** [comment]
Most likely this fork is the future for the LSPosed. The fork from pumPCin has disabled logs. Without logs it is impossible to develop modules and track errors...

**Reply by mywalkb** [reply]
I don't know if it will be the future, however I will work to improve it. I'm alone and don't have as much experience as the original developers, but I try to do my best.

In my mod people can choose. My API or new API, CLI and others.
Only the parasitic manager cannot be disabled at the moment, but I would like to be able to do it, so whoever wants to use the parasitic manager otherwise the classic manager is fine.

**Reply by BlueCat300** [reply]
I don’t understand, the original developers seem to have abandoned development, but why are they still responding in the new API repository as if they were developing?
https://github.com/libxposed/api/issues/15#issuecomment-2002915596

**Reply by mywalkb** [reply]
I don't know, I thought they also archived that repository like others. It's a bit strange behavior.

**Reply by LeeBinder** [reply]
> The fork from pumPCin has disabled logs. Without logs it is impossible to develop modules and track errors...

have you tried his debug releases yet? Like https://github.com/pumPCin/LSPosed_mod/actions/runs/8592473549/artifacts/1392504199 ([LSPosed-v1.9.3-7460-zygisk-debug](https://github.com/pumPCin/LSPosed_mod/actions/runs/8592473549/artifacts/1392504199)) ?

**Reply by juvannx** [reply]
That is the fork of mywalkb, why should we use that fork since the meaning of its changes is not clear? 
Moreover, it does not allow to open issues, which for me is very serious.

**Reply by LeeBinder** [reply]
@juvannx I appreciate your thoughts - 2 x agreed!

**Comment by LeeBinder** [comment]
What are current trustworthy LSPosed forks or mods? I admit I'm out of the loop. Can you please share links? Thank you, folks 👍

**Reply by BD9Max** [reply]
Apparently this is

**Reply by LeeBinder** [reply]
May I finish your sentence for you: ."_Apparently this [mod] is_" not current...

**Reply by BD9Max** [reply]
There is another I don't remember the username currently it has higher version number but only 1 release and version number is probe to be different in forks. 

I just hard reseted my Android and I installed Lsposed by Lsposed. Why? I can update later to a fork that turns out the best fork. 

That other fork is based on mywalkb's so I'm guessing this is likely to have most updates - at least most coding - as one can see from changelog.

**Comment by LeeBinder** [comment]
> That other fork is based on mywalkb's

pumPCin removed his so that can't be it.

> I installed Lsposed by Lsposed

@DB9-max you mean the ancient 1.9.2-7024 (Oct. 11 2023) from https://github.com/LSPosed/LSPosed/releases? If another one, would you please post a link?

**Reply by BD9Max** [reply]
Yes that one but now updated to mywalkb. The other is by jingmatrix. I recommend to use mywalkb's - it continues the legacy of Lsposed with frequent updates unlike other "forks".

**Comment by LeeBinder** [comment]
> I recommend to use mywalkb's - it continues the legacy of Lsposed with frequent updates

The last time mywalkb updated this mod was on [2024-05-25](https://github.com/mywalkb/LSPosed_mod/actions/runs/9234561388#artifacts). That's certainly not "frequent", let alone that update via Actions is not even active anymore... The last release here is from Mar 8, 2024...

> I installed Lsposed by Lsposed. Why? I can update later to a fork that turns out the best fork.
> but now updated to mywalkb.

neither the original nor mywalkb's will give you any "update" options to other "best forks" in the future, that's completely illusional...

> The other is by jingmatrix.

As a matter of fact, that's the first rational and sensible thing you write. I've been using his fork for some days now. Here's all the PROs:

1. Truly updated frequently - last update just a few days ago on [Oct. 16. 2024](https://github.com/JingMatrix/LSPosed/releases/tag/v1.10.1)
2. Even more current updates (right now daily) via [Actions](https://github.com/JingMatrix/LSPosed/actions)
3. Actively maintained [Issue section](https://github.com/JingMatrix/LSPosed/issues): he usually replies in no time and fixes issues if needed
4. [Discussions section](https://github.com/JingMatrix/LSPosed/discussions)
5. Can be installed right over mywalkb's mod

**Reply by BD9Max** [reply]
Yes but mywalkb did some good updating - but jingmatrix seems to be active now. He's one of too commiters to original Lsposed. Apparently this fork he deletes older releases so one might see that he's done only 1 build. Anyhiw glad that Lsposed is alive.

**Comment by LeeBinder** [comment]
Ok. On https://github.com/JingMatrix/LSPosed/releases there is two releases right now.
